testdata/scripts/compose.txt

```diff
@@ -0,0 +1,27 @@
+# Test that the -prog flag works as expected
+
+[!exec:docker-compose] skip 'Requires docker-compose'
+[windows] skip 'TODO: support windows-native images in --volume flags'
+[!windows] env IMAGE=busybox:1.31.1-musl
+
+go test -v -exec 'dockexec -compose eg'
+stdout Hello
+! stderr .+
+
+-- go.mod --
+module mod.com/blah
+
+-- main_test.go --
+package main
+
+import "testing"
+
+func TestThis(t *testing.T) {
+	println("Hello")
+}
+-- docker-compose.yml --
+version: '3.2'
+
+services:
+  eg:
+    image: $IMAGE
```

Comment:
Hm, are you actually testing anything about us selecting the right service? IMO this test would be more interesting if, for example, this service A had access to another service B under a certain hostname and port, so then we could test if we can dial B from the test. That would be a pretty good test, as it should fail under plain `docker run`.

Reply:
> are you actually testing anything about us selecting the right service

As far as I'm aware, yes. Because if we failed to provide a service, or provided one by the wrong/non-existing name then the `docker-compose` call would fail:

```
go test -v -exec 'dockexec -compose eg'
```

I'd tend to favour keeping the test simple, because otherwise we drift somewhat into the realm of testing that `docker-compose` is doing what it should. But let me know what you think.

Reply:
SGTM